django/views/templates/technical_404.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta http-equiv="content-type" content="text/html; charset=utf-8">
   <title>Page not found at {{ request.path_info }}</title>
+  <meta name="color-scheme" content="light dark" />
```

Comment:
```suggestion
  <meta name="color-scheme" content="light" />
```

Since we don’t have support for a dark theme here, we should hard-code so the browser only ever renders a light theme.